lib/view_component.rb

```diff
@@ -14,4 +14,5 @@ module ViewComponent
   autoload :TestCase
   autoload :TemplateError
   autoload :Translatable
+  autoload :Instrumentation
```

Comment:
Maybe keep the `autoload`s in alphabetic order?

Reply:
Good catch!